bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerFragment.java

```diff
@@ -131,16 +131,18 @@ public BookieSocketAddress getAddress(int bookieIndex) {
      * @return entryId
      */
     public long getFirstStoredEntryId() {
```

Comment:
can you please  check the usages of this method ?
I found usages of it as the result was as nullable Long instead of a primitive long

Reply:
hey @eolivelli both in the existing code and the change I made, getFirstStoredEntryId returns primitive long value. I'm not sure what you are referring to.